movieland-web/src/main/java/com/gromoks/movieland/web/controller/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package com.gromoks.movieland.web.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
+    public String handleException(IllegalArgumentException e)
```

Comment:
This also can be configured in .xml. You should send error message in json format

Reply:
can be, but shouldn't we want to escape using xml, especially in @Controllerpart